src/Types.hs

```diff
@@ -160,24 +161,39 @@ data IdeaVoteValue = Yes | No | Neutral
 
 instance SOP.Generic IdeaVoteValue
 
-data IdeaResult = IdeaResult
-    { _ideaResultMeta   :: MetaInfo IdeaResult
-    , _ideaResultValue  :: IdeaResultValue
+data IdeaJuryResult = IdeaJuryResult
+    { _ideaJuryResultMeta   :: MetaInfo IdeaJuryResult
+    , _ideaJuryResultValue  :: IdeaJuryResultValue
     }
   deriving (Eq, Ord, Show, Read, Generic)
 
-instance SOP.Generic IdeaResult
+instance SOP.Generic IdeaJuryResult
 
-data IdeaResultValue
+data IdeaJuryResultValue
     = NotFeasible { _ideaResultNotFeasibleReason :: Document }
     | Feasible    { _ideaResultFeasibleReason    :: Maybe Document }
-    | Winning     { _ideaResultCreatorStatement  :: Maybe Document }
-    | NotEnoughVotes
   deriving (Eq, Ord, Show, Read, Generic)
 
-type instance Proto IdeaResult = IdeaResultValue
+type instance Proto IdeaJuryResult = IdeaJuryResultValue
 
-instance SOP.Generic IdeaResultValue
+instance SOP.Generic IdeaJuryResultValue
+
+data IdeaVoteResult = IdeaVoteResult
+    { _ideaVoteResultMeta   :: MetaInfo IdeaVoteResult
+    , _ideaVoteResultValue  :: IdeaVoteResultValue
+    }
+  deriving (Eq, Ord, Show, Read, Generic)
+
+instance SOP.Generic IdeaVoteResult
+
+data IdeaVoteResultValue
+    = Winning     { _ideaResultCreatorStatement  :: Maybe Document }
+    | EnoughVotes
```

Comment:
Should we add a `Bool` here?  Then we can give all ideas that finish the voting phase a result, instead of marking losing ideas with `Nothing`.  I think I'll do that.